src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -84,6 +91,8 @@ public function setInputStream($stream)
     /**
      * Returns the helper's input stream.
      *
+     * @deprecated Will be removed in 4.0, use StreamableInputInterface::getStream() instead
```

Comment:
Because we have a call to this getter in Application, so lots of tests as deprecations and don't pass anymore.